bgpd/bgp_ecommunity.c

```diff
@@ -998,10 +1060,10 @@ static int ecommunity_lb_str(char *buf, size_t bufsz, const uint8_t *pnt,
 	"rt 100:1 100:2soo 100:3"
 
    extcommunity-list
-	"rt 100:1 rt 100:2 soo 100:3show [ip] bgp" and extcommunity-list regular expression matching
-	"RT:100:1 RT:100:2 SoO:100:3"
+	"rt 100:1 rt 100:2 soo 100:3show [ip] bgp" and extcommunity-list regular
```

Comment:
Since you are touching this part, can you fix this `oo 100:3show [ip] bgp"`? `show [ip] bgp ...` should be on the new line.

Reply:
taken into account